client/me/billing-history/transactions-table.jsx

```diff
@@ -136,13 +136,25 @@ var TransactionsTable = React.createClass( {
 		return description;
 	},
 
+	renderPlaceholder() {
+		return (
+			<tr className="billing-history__transaction is-placeholder">
+				<td className="date">
```

Comment:
Makes sense - it'll be best to also refactor the rest of the code that uses these classnames, so leaving it for a future PR (along with your comment about CSS specificity of selectors).